components/NavigationBar/src/ColorThemer/MDCNavigationBarColorThemer.h

```diff
@@ -14,18 +14,31 @@
  limitations under the License.
  */
 
-#import "MaterialThemes.h"
+#import "MaterialColorScheme.h"
 #import "MaterialNavigationBar.h"
 
 /**
- Used to apply a color scheme to theme MDCNavigationBar.
+ A color themer for MDCNavigationBar that implements the Material design color system mappings.
  */
 @interface MDCNavigationBarColorThemer : NSObject
 
+/**
+ Applies a color scheme's properties to an MDCNavigationBar.
+
+ @param colorScheme The color scheme to apply to MDCNavigationBar.
+ @param navigationBar An MDCNavigationBar instance to which the color scheme should be applied.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                 toNavigationBar:(nonnull MDCNavigationBar *)navigationBar;
+
+#pragma mark - Soon to be deprecated
+
 /**
  Applies a color scheme to theme a MDCNavigationBar. Use a UIAppearance proxy to apply a color
  scheme to all instances of MDCNavigationBar.
 
+ This method will soon be deprecated. Consider using +applySemanticColorScheme:toButtonBar: instead.
```

Comment:
change to "applySemanticColorScheme:toNavigationBar: instead."

Reply:
Done.